src/OrleansZooKeeperUtils/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "OrleansZooKeeperUtils Class Library",
+  "authors": [ "Gutemberg" ],
```

Comment:
I think Gutemberg means into master, we should open an issue on it to track it. This branch is just here so we can collaborate more easily while this is a WIP.